installer/ApplicationSetupUtility.php

```diff
@@ -26,6 +26,14 @@ class ApplicationSetupUtility {
 
     private static $conn;
 
+    /**
+     * Get the log file
+     * @return string
+     */
+    private static function getErrorLogPath() {
+        return realpath(dirname(__FILE__)).'/log.txt';
```

Comment:
use return realpath(dirname(__FILE__)).DIRECTORY_SEPARATOR.'log.txt';
otherwise it may not work in windows server

Reply:
Updated the fix